### ISSUE TITLE
Comparisons with conventional deep learning-based models and supervised training

### ISSUE THREAD

**Issue by JingshuaiLiu** [alanqrwang/hyperrecon#1]
Hi, 

Thanks for the nice paper and code. It provides a novel and interesting pipeline for MRI reconstruction, using regularization weight sampling and a hypernetwork.

I notice that the paper provides no comparison with reconstructions conventionally based on neural networks, i.e. train the U-net to minimize the proposed loss in (3), and subsequently show that the deploy of hypernetworks with data-driven sampling can benefit the de-aliasing practice.

Meanwhile, may I ask if the performance of the proposed method has been tested under supervision, and how it can outperform traditional data-driven methods?

Look forward to your kind reply!

**Comment by alanqrwang** [comment]
Thanks for the comment. 

Regarding your first point, we did compare the performance of our proposed network to a conventional U-Net, individually trained for a grid of hyperparameter settings (see Fig. 3). We show that our proposed model can learn the entire space of regularization settings at the cost of performance due to the amortization penalty.

To your second point, we have not done experiments under supervision, although our method is generally applicable to any hyperparameter in a neural network-based model.
